files/image_config/ntp/mgmtvrf_ntpq

```diff
@@ -0,0 +1,5 @@
+ip link set dev lo master mgmt
+sleep 1
```

Comment:
why do we need this sleep?

Reply:
Immediately after attaching the lo to mgmt vrf,  when we call "ntpq", it fails few times (it succeeds few time). It is consistently successful only with this 1 second sleep.  If we can expect the user to execute the command couple of times to see successful result, this 1 second sleep is not required. If you want us to look into alternate solutions to solve this, let us know.